neuromation/api/parser.py

```diff
@@ -144,6 +145,16 @@ def _remote_to_local_image(self, image: RemoteImage) -> LocalImage:
         )
         return parser.convert_to_local_image(image)
 
+    def env(self, env: Sequence[str]) -> Tuple[Dict[str, str], Dict[str, URL]]:
```

Comment:
Should not it have the `env_file` parameter?

Reply:
Good catch. Added